api/static/rest_framework_swagger/css/osf_screen.css

```diff
@@ -0,0 +1,1232 @@
+/* http://meyerweb.com/eric/tools/css/reset/ v2.0 | 20110126 */
+.swagger-ui-wrap html,
```

Comment:
Instead of overriding the entire CSS file, why not add a new CSS file, and in a *template* override, include the new CSS file?